packages/perseus/src/widgets/interactive-graphs/interactive-graph.test.tsx

```diff
@@ -102,310 +102,318 @@ describe("Interactive Graph", function () {
-                    expect(score).toHaveInvalidInput();
-                });
-            },
-        );
+    const graphQuestionRenderers: {
+        [K in PerseusGraphType["type"][number]]: PerseusRenderer;
+    } = {
+        angle: angleQuestion,
+        segment: segmentQuestion,
+        linear: linearQuestion,
+        "linear-system": linearSystemQuestion,
+        ray: rayQuestion,
+        polygon: polygonQuestion,
+        point: pointQuestion,
+        circle: circleQuestion,
+        quadratic: quadraticQuestion,
+        sinusoid: sinusoidQuestion,
+        "unlimited-point": pointQuestion,
+        "unlimited-polygon": polygonQuestion,
+    };
+
+    const graphQuestionRenderersCorrect: {
+        [K in PerseusGraphType["type"][number]]: PerseusRenderer;
+    } = {
+        angle: angleQuestionWithDefaultCorrect,
+        segment: segmentQuestionDefaultCorrect,
+        linear: linearQuestionWithDefaultCorrect,
+        "linear-system": linearSystemQuestionWithDefaultCorrect,
+        ray: rayQuestionWithDefaultCorrect,
+        polygon: polygonQuestionDefaultCorrect,
+        point: pointQuestionWithDefaultCorrect,
+        circle: circleQuestionWithDefaultCorrect,
+        quadratic: quadraticQuestionWithDefaultCorrect,
+        sinusoid: sinusoidQuestionWithDefaultCorrect,
+        "unlimited-point": pointQuestionWithDefaultCorrect,
+        "unlimited-polygon": polygonQuestionDefaultCorrect,
+    };
+
+    describe.each(Object.entries(graphQuestionRenderers))(
+        "graph type %s",
+        (_type, question) => {
+            it("should render", () => {
+                renderQuestion(question, blankOptions);
+            });
 
-        describe.each(Object.entries(graphQuestionRenderersCorrect))(
-            "graph type %s: default correct",
-            (_type, question) => {
-                it("should render", () => {
-                    renderQuestion(question, blankOptions);
-                });
+            it("should render when the correct answer is not present", () => {
+                // As part of implementing server-side scoring (Q1 2025) we are
+                // removing answers from the widget data that's initially sent
+                // to the frontend. This test ensures that interactive graphs
+                // can render when the answers have been stripped out of the
+                // data.
 
-                // TODO(jeremy): This test is disabled because it fails
-                // sporadically (especially on slower/lower-end computers, like
-                // CI). Will work on a fix after the React 18 release.
-                it.skip("rejects incorrect answer", async () => {
-                    // Arrange
-                    const {renderer} = renderQuestion(question, blankOptions);
+                const answerlessQuestion = splitPerseusItem(question);
 
-                    await userEvent.tab();
+                renderQuestion(answerlessQuestion, blankOptions);
```

Comment:
We don't need an assertion? Just rendering?

Reply:
None of the other interactive graph tests check for specific elements being rendered. They're just testing that rendering doesn't throw an error. I didn't want to shave that yak in this PR, so what I decided to do instead is add a visual regression test in Storybook to make sure the elements render. We have several such tests for Interactive Graph already.